src/style.css

```diff
@@ -1,3 +1,73 @@
+@font-face {
+  font-family: Roboto;
+  src: url(font/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(font/Roboto-Medium.ttf);
+  font-weight: 500;
+}
+
 body {
+  --color-active: #00acdc;
+
+  margin: 0;
+  background-color: #ddd;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
   margin: 0;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  position: relative;
+  text-decoration: none;
+  font-weight: 500;
+  color: #000;
+  font-size: 12px;
+  font-family: Roboto, Helvetica, sans-serif;
```

Comment:
It's a good practice to include a fallback font-family in case the main one doesn't work. For instance, 'font-family: Roboto, Arial, sans-serif;'.